ui/app/scripts/app/services.js

```diff
@@ -73,10 +74,15 @@ define(['angular', 'lodash'], function (angular, _) {
                     return request;
                 },
                 getAppInfo: function(appType,appName) {
-                    return $http({
-                        method: 'GET',
-                        url: $rootScope.dataflowServerUrl + '/apps/'+appType+'/'+appName
-                    });
+                    return $resource($rootScope.dataflowServerUrl + '/apps/'+appType+'/'+appName, {}, {
+                        query: {
+                            method: 'GET'
+                        }
+                    }).get();
+                    // return $http({
```

Comment:
Please remove the commented out lines (on this PR) unless they are really needed for references.